client-app/modules/purchase-requests/composables/usePurchaseRequestDocuments.ts

```diff
@@ -0,0 +1,41 @@
+import { ref } from "vue";
+import { DEFAULT_FILES_SCOPE } from "@/modules/purchase-requests/constants";
+import { useFiles } from "@/shared/files/composables/useFiles";
+import { isUploadedFile } from "@/ui-kit/utilities";
+import type { WatchSource } from "vue";
+
+export function usePurchaseRequestDocuments(sourceFiles?: WatchSource<IAttachedFile[]>) {
+  const {
+    files,
+    addFiles,
+    validateFiles,
+    uploadFiles,
+    fetchOptions: fetchFileOptions,
+    options: fileOptions,
+  } = useFiles(/* config.purchase_requests_file_scope ?? */ DEFAULT_FILES_SCOPE, sourceFiles);
```

Comment:
file scope in module settings now. see quotes

Reply:
done